.github/workflows/pull-request-checks.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
       - name: Get minisat
         run: make -C lib/cbmc/src minisat2-download
       - name: Build with make
-        run: make CXX="ccache g++" -C src -j2
+        run: make CXX="ccache g++ -lstdc++fs -Wno-class-memaccess" -C src -j2
```

Comment:
I believe this needs to be `make CXX="ccache g++ -Wno-class-memaccess" LIBS="-lstdc++fs" -C src -j2` to work.